docs/application/native/guides/text-input/autofill-service.md

```diff
@@ -0,0 +1,332 @@
+        autofill_fill_response_item_set_value(res_it_h[1], "testerpw2");
+
+        /* Add item 2 in group 2 */
+        autofill_fill_response_group_add_item(res_group_h[1], res_it_h[1]);
+
+        autofill_fill_response_item_destroy(res_it_h[0]);
+        autofill_fill_response_item_destroy(res_it_h[1]);
+
+        autofill_fill_response_add_group(fill_response_h, res_group_h[1]);
+
+        autofill_fill_response_group_destroy(res_group_h[1]);
+
+        /* Send fill response */
+        autofill_service_send_fill_response(context_id, fill_response_h);
+
+        autofill_fill_response_destroy(fill_response_h);
+    }
+
+    bool service_app_create(void *data)
+    {
+        autofill_service_initialize();
+
+        ...
+
+        autofill_service_set_fill_requested_cb(_fill_request_received_cb, NULL);
+
+        ...
+
+        return true;
+    }
+    ```
+
+## Receive Request of Saving Autofill Data
+
+1. To receive the request of saving Autofill data, create the callback function using the `void _commit_received_cb()` function:
+
+    ```c
+    static void _commit_received_cb(int context_id, autofill_save_view_info_h vi_h, void *user_data)
+    {
+        char *view_id = NULL;
+        autofill_save_view_info_get_view_id(vi_h, &view_id);
+        dlog_print(DLOG_INFO, LOG_TAG, "view id : %s", view_id);
+
+        autofill_save_view_info_foreach_item(vi_h, __save_item_cb, NULL);
+
+        if (view_id)
+            free(view_id);
+    }
+
+    bool service_app_create(void *data)
+    {
+        autofill_service_initialize();
+
+        ...
+        autofill_service_set_commited_cb(_commit_received_cb, NULL);
+        ...
+
+        return true;
+    }
+    ```
+
+2. Store the autofill data received by the callback, in a secure storage:
+
+    ```c
+    static bool __save_item_cb(autofill_save_item_h it_h, void *user_data)
+    {
+        char *id = NULL;
+        char *label = NULL;
+        char *value = NULL;
+        autofill_hint_e autofill_hint;
+        bool sensitive_data;
+
+        autofill_save_item_get_id(it_h, &id);
+        dlog_print(DLOG_INFO, LOG_TAG, "id : %s", id);
+
+        autofill_save_item_get_label(it_h, &label);
+        dlog_print(DLOG_INFO, LOG_TAG, "label : %s", label);
+
+        autofill_save_item_get_value(it_h, &value);
+        dlog_print(DLOG_INFO, LOG_TAG, "value : %s", value);
+
+        autofill_save_item_get_autofill_hint(it_h, &autofill_hint);
+        dlog_print(DLOG_INFO, LOG_TAG, "autofill hint : %d", autofill_hint);
+
+        autofill_save_item_get_sensitive_data(it_h, &sensitive_data);
+        dlog_print(DLOG_INFO, LOG_TAG, "sensitive data : %d", sensitive_data);
+
+        /* Store the autofill data in your secure storage */
+
+        free(id);
+        free(label);
+        free(value);
+
+        return true;
+    }
+    ```
+
+## Release Resources
+
+After you have finished working with the Autofill Service library, de-initialize the Autofill Service:
```

Comment:
de-initialize is a single word.

Reply:
de-initialize -> deinitialize
I've fixed.